.github/workflows/sign-extension.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       - name: Upload artifact
         uses: actions/upload-artifact@v4
         with:
-          name: signed-plugin
+          name: plainly-plugin
```

Comment:
i think name is optional and if we leave it out it will just have the file directly, can we try that?

Reply:
no, it returns `artifact.zip`